modules/ROOT/pages/_partials/sync/js-client-getting-started.adoc

```diff
@@ -1,10 +1,10 @@
 = Getting Started
 
-This describes how to start using JavaScript library for creating mobile and web applications using {sync-service} provided by Voyager Server. 
+This section describes how to use the Voyager JavaScript client SDK to create mobile and web applications that can communicate with the Voyager server application. 
 
 .Prerequisites
 
-* Your local `mobile-services.json` file is up-to-date.
```

Comment:
I think that we have a "split personality" in the documentation. So far the documentation is very much counting on that we use MDC and `mobile-services.json` and have it in OpenShift. It is possible to use other services independently as well, we should maybe remove it from other sections too.

Reply:
not sure i get what you mean, this document isn't really counting on using `mobile-services.json`. I changed it say just make sure there is a server running.